src/toPairs.test.ts

```diff
@@ -25,7 +25,7 @@ test('stricter typing', () => {
 
 test('stricter typing with optional', () => {
   const actual = toPairs.strict({} as { a?: string });
-  assertType<Array<['a', string]>>(actual);
+  assertType<Array<['a', string | undefined]>>(actual);
```

Comment:
I think is correct that the undefined is part of the value type of the tuple

Reply:
I don't think so, If `a` is just optional, but not nullable, when will `Object.toEntries()` ever return `[['a', undefined]]`?